src/main/java/uk/gov/companieshouse/web/accounts/controller/corporationtax/CorporationTaxController.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.companieshouse.web.accounts.controller.corporationtax;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.companieshouse.web.accounts.annotation.NextController;
+import uk.gov.companieshouse.web.accounts.controller.BaseController;
+import uk.gov.companieshouse.web.accounts.controller.smallfull.CriteriaController;
+import uk.gov.companieshouse.web.accounts.model.corporationtax.CorporationTax;
+
+import javax.validation.Valid;
+
+@Controller
+@NextController(CriteriaController.class)
+@RequestMapping("/company/{companyNumber}/corporation-tax")
+public class CorporationTaxController extends BaseController {
+
+    @GetMapping
+    public String getCorporationTax(Model model) {
+
+        model.addAttribute("corporationTax", new CorporationTax());
+
+        return getTemplateName();
+    }
+
+    @PostMapping
+    public String postCorporationTax(@PathVariable String companyNumber,
+                                     @ModelAttribute("corporationTax") @Valid CorporationTax corporationTax,
+                                     BindingResult bindingResult) {
+
+        if (bindingResult.hasErrors()) {
+            return getTemplateName();
+        }
+
+        if (corporationTax.getFileChoice().equals(true)) {
```

Comment:
`.equals` not required here; could just use `if (corporationTax.getFileChoice())`. Could the name of the boolean on the model be more descriptive, like `fileWithHMRC`?